Documentation/development-environment.md

```diff
@@ -12,13 +12,24 @@ There are a bunch of pre-requisites to be able to deploy the following environme
 
 * A Linux workstation (CentOS or Fedora)
 * KVM/QEMU installation
+* docker service allowing insecure local repository
 
 For other Linux distribution, there is no guarantee the following will work.
 However adapting commands (apt/yum/dnf) could just work.
 
 ## Prerequisites installation
 
-On your host machine, execute `tests/scripts/multi-node/rpm-system-prerequisites.sh`.
+On your host machine, execute `tests/scripts/multi-node/rpm-system-prerequisites.sh` (or
+do the equivalent for your distribution)
+
+Edit `/etc/docker/daemon.json` to add insecure-registries:
```

Comment:
Why don't we append this line instead with `sed` (or something else)? Instead of requiring a manual step?

Reply:
People may already have a daemon.json, and indeed an insecure-registries key in it; also, this is intrusive enough that I wanted to make clear it's happening, so people can choose to set up their own private container/VM if they wish to avoid all the host pollution

Reply:
Fair enough.